server/bot_api.go

```diff
@@ -341,12 +341,13 @@ func (p *Plugin) handleBotPostTranscriptions(w http.ResponseWriter, r *http.Requ
 		UserId:    p.getBotID(),
 		ChannelId: callID,
 		Message:   postMsg,
-		Type:      "custom_calls_transcription",
+		Type:      callTranscriptionType,
 		RootId:    threadID,
 		FileIds:   []string{info.Transcriptions[0].FileIDs[1]},
 	}
 	transcriptionPost.AddProp("call_post_id", info.PostID)
 	transcriptionPost.AddProp("transcription_id", info.JobID)
+	transcriptionPost.AddProp("web_vtt_file_id", info.Transcriptions[0].FileIDs[0])
```

Comment:
Added because it was easier than hunting it down with the other props. 0/5 if there is a another preferred way.

Reply:
Is there a fundamental difference in whether we feed to the model the VTT captions file vs the text format? Data should be pretty much the same. Is the former easier to deal with prompting wise perhaps?

Reply:
I have some code that parses the VTT from whisper and formats it better for the LLM. I didn't want to have to parse the custom text format to replicate.

Reply:
Fair enough. In that case would you mind adding the whole `captions` prop instead, same as we do for the recording post?

Reply:
@streamer45 Done. I guess the plan with that prop is to support captions in multiple languages?

Reply:
Thanks. And yes eventually we could be doing automated translations so I kept it open to that. For now it will be just one, in whatever language was detected while transcribing.